package/gluon-core/luasrc/usr/lib/lua/gluon/platform.lua

```diff
@@ -55,4 +56,33 @@ function M.is_outdoor_device()
 	return false
 end
 
+function M.device_supports_wpa3()
+	if M.match('ramips', 'rt305x') or M.match('ramips', 'mt7620') then
```

Comment:
Is it possible to add a reference here why those exact targets are not supporting WPA3? How did you identify they don't?

Reply:
done